src/utils/daos_autotest.c

```diff
@@ -714,12 +776,24 @@ static int
 cdestroy(void)
 {
 	int rc;
-
 	rc = daos_cont_destroy(poh, cuuid, force, NULL);
 	if (rc) {
 		step_fail(d_errdesc(rc));
 		return -1;
 	}
+
+	rc = daos_cont_destroy(poh, cuuid2, force, NULL);
+	if (rc) {
+		step_fail(d_errdesc(rc));
+		return -1;
+	}
+
+	rc = daos_cont_destroy(poh, cuuid3, force, NULL);
+	if (rc) {
+		step_fail(d_errdesc(rc));
+		return -1;
+	}
```

Comment:
I am not sure what is the point of creating 2 extra containers with RF1 and 2 and not using them.

Reply:
For RF1 and RF2 insert and read tests have been implemented.